plugwise_usb/nodes/sense.py

```
@@ -90,35 +88,50 @@
-            self._humidity = int(
+            self._sense_statistics.humidity = float(
                 SENSE_HUMIDITY_MULTIPLIER * (response.humidity.value / 65536)
                 - SENSE_HUMIDITY_OFFSET
             )
             await self.publish_feature_update_to_subscribers(
-                NodeFeature.HUMIDITY, self._humidity
+                NodeFeature.SENSE, self._sense_statistics
             )
-            report_received = True
+            report_received_2 = True
     
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)